llvm-runtimes/libunwind/libunwind-20.1.7.ebuild

```diff
@@ -4,8 +4,8 @@
 EAPI=8
```

Comment:
You meant 20.1.8, but I guess it's easy to sed it while applying the patch.

Reply:
Right, sorry, I will skip 20.1.7 and apply just to 20.1.8.

I hope you are fine with changing 19.1.7. Currently, musl-llvm stage3 and crossdev are stuck on LLVM 19. I'm trying to fix both issues:

* https://github.com/gentoo/gentoo/pull/42663 (I need to apply your suggestion about building the config partially, I'll try to get it done this week)
* https://github.com/gentoo/gentoo/pull/42712 (I think this one shouldn't require any more work)

If you prefer to land these two PRs beforehand and trim this one down just to LLVM 20, that's fine with me.